Common/DTO/UserCredential.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace DTO
+{
+    /// <summary>
+    /// Minimal model of user: contains email and password
+    /// </summary>
+    public class UserCredential
+    {
+        public UserCredential(string email, string password)
```

Comment:
It is a good practice to put class methods after class fields and class properties

Reply:
fixed